src/io/reader.cpp

```diff
@@ -185,53 +188,109 @@ las::Points Reader::GetAllPoints()
     return out;
 }
 
-std::vector<Node> Reader::GetNodesWithinBox(const Box &box)
+std::vector<Node> Reader::GetNodesWithinBox(const Box &box, double min_resolution)
```

Comment:
these should be `max_resolution`

Reply:
Ha that's confusing, since the value itself is the minimum value?